_adr/81/index.adoc

```diff
@@ -19,10 +19,243 @@ NOTE: https://datatracker.ietf.org/doc/html/rfc2119[RFC-2119] keywords are used
+          "parameters": {
+            "ingress": "90MB/s",
+            "egress": "90MB/s",
+            "partitions": "3000",
+            "connections": "1500",
+            "connectionRate": "100",
+            "maxMessageSize": "1MB"
+          }
+        }
+      ],
+    },
+    // other series as necessary
+]
+----
+<1> This list is not considered ordered
+<2> Names MUST be unique within the `series` list 
+<3> The `id` uniquely identifies a series, allowing the name to be changed
+<4> Can be used in a UI/CLI to help guide users to appropriate profile/flavour of the service
+<5> The list MUST be ordered such that later items represent clusters with more capacity
+<6> The name MUST be unique within the `sizes` list
+<7> The parameters MUST be consistent within a series, but different series MAY have different parameters.
+
+NOTE: The above is not intended as a detailed API specification, merely an illustration of how the series and size are communicate from the Fleet Manager to a UI.
+
+NOTE: It may be necessary, eventually, to distinguish the supported actions that are supported for a given series and size.
+For example, we might want to prevent the creation of new instances of the "ex1" series while supporting existing instances. 
+So we might eventually want additional properties alongside "name" and "parameters" in the instance size schema.
+
+==== Instance creation
+
+The `POST /api/kafkas_mgmt/v1/kafkas` endpoint would change to take the series id and size name in addition to the existing parameters.
+
+[source,json]
+.Example `POST /api/kafkas_mgmt/v1/kafkas` made by a UI to the Fleet Manager to create an instance
+----
+{
+    "region": "us-east-1",
+    "cloud_provider": "aws",
+    "seriesId": 1, <1>
+    "sizeName": "small", <2>
+    "name": "serviceapitest"
+
+}
+----
+<1> New property, the id refers to the id of the series previously served from the series discovery endpoint
+<2> New property, the name refers to the name of the size in the given series.
+
+To allow the API to evolve compatibly, the manager is allowed to use a default series and size in the case that the `POST` request omits these properties.
+
+If the requested series and/or size is not available in that cloud provider and region a HTTP 400 error response is returned.
+
+==== Getting instance state
+
+Similarly, the `GET /api/kafkas_mgmt/v1/kafkas/{id}` endpoint would change to include the series and size.
+
+=== Fleet manager responsibilities
+
+The supported series are passed to the fleet manager via app interface.
+
+When terraforming a shard, the manager needs to keep track of which series are/will be supported on that shard.
+
+Currently it is acceptable for the manager to embed knowledge of the machine types needed by the shard.
+The manager does not know how the nodes running on those machines will be used.
+It is expected that a future ADR will describe a mechanism for the shard and manager to dynamically adjust the number of nodes.
```

Comment:
I think it is reasonable to keep it vague since we are saying it will be described in a future ADR. What do you feel is missing here that could add context to this ADR?

Reply:
I guess I wanted some more formal wording to the effect that "The Fleet Manager does the business stuff, the Shard does the provisioning implementation stuff". 

I'm happy if you and the fleet manager people are happy with the responsibilities, both called out and implied, in this ADR.